Modules/StudyProgramme/classes/class.ilObjStudyProgrammeSettingsGUI.php

```diff
@@ -269,7 +279,65 @@ protected function buildForm(\ilObjStudyProgramme $prg, string $submit_action) :
 		}));
 	}
 
-	protected function updateWith(\ilObjStudyProgramme $prg, array $data) {
+
+	protected function getDeadlineSubform($prg)
+	{
+		$ff = $this->input_factory->field();
+		$txt = function($id) { return $this->lng->txt($id); };
+		$deadline_period_subform = $ff->numeric('',$txt('prg_deadline_period_desc'))
+										->withAdditionalConstraint(
+											$this->validation->greaterThan(-1)
+									);
+		$period = $prg->getDeadlinePeriod();
+		$radion_option = self::OPT_NO_DEADLINE;
```

Comment:
Was ist ein radion? =)

Reply:
Nunja:

Radion

Hypothetisches Kraftfeld, das von dem Kosmologen Paul Steinhardt eingeführt. Das Radion durchsetze die ganze Raumzeit im Kosmos. Als Folge dieses Feldes manifestiere sich das, was in der Kosmologie als Dunkle Energie bezeichnet wird. Das fluktuierende, zeitlich variable Radionfeld solle dafür verantwortlich sein, dass unser Vorläuferuniversum mit einem Paralleluniversum kollidiert sei. Aus dieser Weltenkollision, der so genannten Ekpyrosis, sei schließlich – so das sehr spekulative Modell – unser heutiges Universum hervorgegangen!
Natürlich gibt es für diese Idee keinerlei Stützen auf der Grundlage astronomischer Beobachtungen. Allerdings ist eine attraktive Eigenschaft dieser Hypothese, dass sie erstmals eine Erklärung für den Urknall anbietet – das vermag die Urknalltheorie nicht.

Reply:
Ich finde, es gibt genug "fluktuierende, zeitlich variable" Felder in ILIAS, und ausreichend Dunkle Energie. Da braucht's nicht noch mehr von...